extensions/vscode/webviews/homeView/src/components/views/secrets/Secrets.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <TreeSection
+    title="Secrets"
+    :actions="sectionActions"
+    :count="home.secretsWithValueCount"
+  >
+    <template v-if="home.secrets.size">
+      <Secret v-for="[name] in home.secrets" :name="name" :key="name" />
+    </template>
+    <WelcomeView v-else>
+      <p>No secrets have been added to the configuration.</p>
+    </WelcomeView>
+  </TreeSection>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { ActionButton } from "src/components/ActionToolbar.vue";
+import TreeSection from "src/components/tree/TreeSection.vue";
+import WelcomeView from "src/components/WelcomeView.vue";
+import { useHomeStore } from "src/stores/home";
+import Secret from "src/components/views/secrets/Secret.vue";
+
+const home = useHomeStore();
+
+const sectionActions = computed<ActionButton[]>(() => [
+  {
+    label: "Clear Values for all Secrets",
+    codicon: "codicon-clear-all",
+    fn: () => {
+      home.secrets.forEach((_, key) => {
+        home.secrets.set(key, undefined);
+      });
+    },
+  },
```

Comment:
I could optionally show this only if 1 or more secrets have values?

Reply:
The more I use this the more I want to do this, since the button moves if the count badge is showing or not.